treeherder/credentials/models.py

```diff
@@ -14,6 +14,6 @@ class Credentials(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     modified = models.DateTimeField(auto_now=True)
 
-    class Meta:
+    class Meta(object):
```

Comment:
I don't think there's anything wrong with this… but it's not something I've ever come across before!  Even Django doesn't do this in their contrib apps.  It causes a lot of noise in the diff and looks different to lots of other Django projects… do we need it?

Reply:
For the `Meta` classes specifically? Ah yeah they don't:
https://github.com/django/django/blob/1.11.12/django/contrib/admin/models.py#L63

Agreed makes sense to just drop this change for `Meta`, since the Python 2.x differences between `object` and not presumably aren't applicable for this minimal class.